pages/common/atom.md

```diff
@@ -0,0 +1,11 @@
+# atom
+
+> A hackable text editor for the 21st century.
```

Comment:
Uh, I guess I meant both, but yeah that is silly. How about just "A cross-platform text editor."